### PR TITLE
Sort on user committees and add colors

### PR DESCRIPTION
**Pull request by JeremieSamson**

### CODE REVIEW

File: src/Collection/CommitteeCollection.php [hunk 1]
```diff
@@ -24,10 +24,16 @@ public function getOrderedCommittees(CommitteeMembershipCollection $memberships)
             return in_array((string) $committee->getUuid(), $uuids);
         });
 
+        $uuids = $memberships->getCommitteeUnApprovedMemberships(CommitteeMembershipCollection::EXCLUDE_SUPERVISORS)->getCommitteeUuids();
```
Comment (Remg) [inline src/Collection/CommitteeCollection.php]: is the argument useless or not handled in the method ?
`public function getCommitteeUnApprovedMemberships(): self`

File: src/Collection/CommitteeMembershipCollection.php [hunk 2]
```diff
@@ -66,7 +66,7 @@ public function getCommitteeHostMemberships(int $withSupervisors = self::INCLUDE
     public function getCommitteeFollowerMemberships(): self
     {
         return $this->filter(function (CommitteeMembership $membership) {
-            return $membership->isFollower();
+            return $membership->isFollower() && !$membership->getCommittee()->isPending();
```
Comment (thomasbisignani) [inline src/Collection/CommitteeMembershipCollection.php]: What do you think to rename the `getCommitteeFollowerMemberships`method to `getCommitteeFollowerMembershipsInProcess` ?

File: src/Collection/CommitteeMembershipCollection.php [hunk 3]
```diff
@@ -77,6 +77,13 @@ public function getCommitteeSupervisorMemberships(): self
         });
     }
 
+    public function getCommitteeUnapprovedMemberships(): self
```
Comment (thomasbisignani) [inline src/Collection/CommitteeMembershipCollection.php]: What do you think to rename the `getCommitteeUnapprovedMemberships`method to `getCommitteeMembershipsInPending` ?

Comment (illusionOfParadise) [reply src/Collection/CommitteeMembershipCollection.php]: @JeremieSamson Are you sure that we need a list of only pending committees without pre-approved and pre-refused ones?

File: src/Collection/CommitteeMembershipCollection.php (hunk 2 above)
Comment (illusionOfParadise) [reply src/Collection/CommitteeMembershipCollection.php]: @thomasbisignani the actual name seems good. It's the opposite, we check that the committee status is not pending.

File: src/Collection/CommitteeMembershipCollection.php (hunk 3 above)
Comment (Remg) [reply src/Collection/CommitteeMembershipCollection.php]: is it confirmed ? @JeremieSamson

**Pull request merged**